### PR TITLE
chore: Agenda Detail Layout

### PR DESCRIPTION
**Pull request by Ibwedagama**
#### Related
- [S27A2 - View Detail Agenda
](https://airtable.com/app7SdZInMN1pG6ZL/tblLy5A3qYF19fj1u/viwDUthwzD6mc5jka/rec2uPTynFS1JzKqQ?blocks=hide)

#### Changes
- add `BaseButton` component
- add `HeaderMenu` component
- add `AgendaDetailPage`, `AgendaDetail` component
- add `import resolver` setting on `eslintrc.js`

#### Preview
Screen Shot 2022-01-12 at 16 01 13

### Evidence
title: agenda detail layout
project: Portal Jabar
participants: @Ibwedagama @bangunbagustapa @maulanayuseph @naufalihsank

### CODE REVIEW

File: src/router/routes.js [hunk 1]
```diff
@@ -14,6 +14,11 @@ export default [
     name: 'Agenda Jawa Barat',
     component: () => import('@/pages/AgendaPage'),
   },
+  {
+    path: '/agenda/detail',
+    name: 'Agenda Jawa Barat',
+    component: () => import('@/pages/AgendaDetailPage.vue'),
```
Comment (bangunbagustapa) [inline src/router/routes.js:20]: I think you should remove the file extention to be more consistent?

File: src/components/Agenda/AgendaDetail.vue [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        Detail Agenda Jawa Barat
+      </h1>
+      <!-- TODO: Add page content -->
+    </section>
+  </main>
+</template>
+
+<script>
+import BaseButton from '../ui/BaseButton.vue';
+import HeaderMenu from '../ui/HeaderMenu.vue';
```
Comment (bangunbagustapa) [inline src/components/Agenda/AgendaDetail.vue:31]: In other components, we import the component with path alias. I think we should stick to one way to import/ What do you think?

File: src/router/routes.js (hunk 1 above)
Comment (Ibwedagama) [reply src/router/routes.js:20]: I tried earlier, but I don't know it's always says something like this:
Screen Shot 2022-01-13 at 09 54 14

any idea how to solve this? @bangunbagustapa

File: src/components/Agenda/AgendaDetail.vue (hunk 2 above)
Comment (Ibwedagama) [reply src/components/Agenda/AgendaDetail.vue:31]: oh my bad I think it's my auto import extensions 🙏🏼 

will fix it asap @bangunbagustapa

Comment (Ibwedagama) [reply src/components/Agenda/AgendaDetail.vue]: I fix it [here](https://github.com/jabardigitalservice/portal-jabar-cms/pull/23/commits/ca14a78e61e1d247638e8d0d0503b52fbfe49db6) thanks

File: src/router/routes.js (hunk 1 above)
Comment (Ibwedagama) [reply src/router/routes.js]: solved on this [commit](refactor (route): remove vue extension on agenda-detail-page route)

**Review by bangunbagustapa — APPROVED** [review]

**Review by naufalihsank — APPROVED** [review]

**Review by maulanayuseph — APPROVED** [review]

**Pull request merged**